index.html

```diff
@@ -1,2 +1,107 @@
+<!DOCTYPE html>
 <html>
+<head>
+    <title>Kate Starks</title>
+    <link rel='stylesheet' type='text/css' href='css/normalize.css'/>
+    <link rel='stylesheet' type='text/css' href='css/styles.css'/>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:300,500" rel="stylesheet">
+</head>
+
+<body>
+    <main>
+        <div class="hero">
+            <h1 class="heroHeading">Kate Starks {</h1>
+            <p class="heroPara">Full Stack Developer</p>
+            <p class="heroPara">Positive | Resilient | Motivated</p>
+            <h1 class="heroHeading">}</h1>
+            <div><a href="#about"><img class="heroArrow" src="IMG/heroArrow.png"></a></div>
+        </div>
+    </main>
+    <nav>
+        <ul>
+            <li><a href="#about">About</a></li>
+            <li><a href="#projects">Projects</a></li>
+            <li><a href="#contact">Contact me</a></li>
+        </ul>
+        <div class="navColor"></div>
+    </nav>
+    <section id="about" class="about">
+        <div class="aboutContent aboutContentLeft">
+            <h1>" Welcome to my portfolio,</h1>
+            <p>Hi, I’m Kate, a full stack developer in training. Learning to code is a career change for me. I was looking for a tangible, vocational skill to complement the ‘soft’ skills I’ve picked up. Someone told me that once you know how to code, the only limit is your imagination.</p>
+            <p class="contentEmphasisLine">I was captivated by that idea and, now that I’m learning, I’m hooked on the process.<span class="contentQuote">"</span></p>
+        </div>
+        <div class="aboutContent aboutContentRight">
```

Comment:
same again about class declaration, can just use .about div {} instead of .aboutContent